randobot/handler.py

```diff
@@ -783,6 +796,36 @@ async def ex_presetsdev(self, args, message):
             return
         await self.send_presets(True)
 
+    async def ex_password(self, args, message): 
```

Comment:
There's no protection on this command, in the docs you say it's monitor/moderator only, in which case you want the `@monitor_cmd` decorator here